src/lib/util/Timestamp.js

```diff
@@ -153,114 +266,20 @@ class Timestamp {
-	const diff = ((time.getMilliseconds() - start.getMilliseconds()) + (start.getTimezoneOffset() - time.getTimezoneOffset())) * MINUTE;
-	return String(Math.floor(diff / DAY));
-};
-
-Timestamp.d = time => {
-	const day = String(time.getDate());
-	if (day !== '11' && day.endsWith('1')) return `${day}st`;
-	if (day !== '12' && day.endsWith('2')) return `${day}nd`;
-	if (day !== '13' && day.endsWith('3')) return `${day}rd`;
-	return `${day}th`;
-};
-
-Timestamp.dd = time => DAYS[time.getDay()].slice(0, 2);
-
-Timestamp.ddd = time => DAYS[time.getDay()].slice(0, 3);
-
-Timestamp.dddd = time => DAYS[time.getDay()];
-
-Timestamp.X = time => String(time.valueOf() / SECOND);
-
-Timestamp.x = time => String(time.valueOf());
-
-// Times
-
-Timestamp.H = time => String(time.getHours());
-
-Timestamp.HH = time => String(time.getHours()).padStart(2, '0');
-
-Timestamp.h = time => String(time.getHours() % 12 || 12);
-
-Timestamp.hh = time => String(time.getHours() % 12 || 12).padStart(2, '0');
-
-Timestamp.a = time => time.getHours() < 12 ? 'am' : 'pm';
-
-Timestamp.A = time => time.getHours() < 12 ? 'AM' : 'PM';
-
-Timestamp.m = time => String(time.getMinutes());
-
-Timestamp.mm = time => String(time.getMinutes()).padStart(2, '0');
-
-Timestamp.s = time => String(time.getSeconds());
-
-Timestamp.ss = time => String(time.getSeconds()).padStart(2, '0');
-
-Timestamp.S = time => String(time.getMilliseconds());
-
-Timestamp.SS = time => String(time.getMilliseconds()).padStart(2, '0');
-
-Timestamp.SSS = time => String(time.getMilliseconds()).padStart(3, '0');
-
-// Locales
-
-/* eslint max-len:0 new-cap:0 */
-
-Timestamp.T = (time) => `${Timestamp.h(time)}:${Timestamp.mm(time)} ${Timestamp.A(time)}`;
-
-Timestamp.t = (time) => `${Timestamp.h(time)}:${Timestamp.mm(time)}:${Timestamp.ss(time)} ${Timestamp.A(time)}`;
-
-Timestamp.L = (time) => `${Timestamp.MM(time)}/${Timestamp.DD(time)}/${Timestamp.YYYY(time)}`;
-
-Timestamp.l = (time) => `${Timestamp.M(time)}/${Timestamp.DD(time)}/${Timestamp.YYYY(time)}`;
-
-Timestamp.LL = (time) => `${Timestamp.MMMM(time)} ${Timestamp.DD(time)}, ${Timestamp.YYYY(time)}`;
-
-Timestamp.ll = (time) => `${Timestamp.MMMM(time).slice(0, 3)} ${Timestamp.DD(time)}, ${Timestamp.YYYY(time)}`;
-
-Timestamp.LLL = (time) => `${Timestamp.LL(time)} ${Timestamp.T(time)}`;
-
-Timestamp.lll = (time) => `${Timestamp.ll(time)} ${Timestamp.T(time)}`;
-
-Timestamp.LLLL = (time) => `${Timestamp.dddd(time)}, ${Timestamp.LLL(time)}`;
-
-Timestamp.llll = (time) => `${Timestamp.ddd(time)} ${Timestamp.lll(time)}`;
-
-Timestamp.Z =
-Timestamp.ZZ = time => {
-	const offset = time.getTimezoneOffset();
-	return `${offset >= 0 ? '+' : '-'}${String(offset / -60).padStart(2, '0')}:${String(offset % 60).padStart(2, '0')}`;
-};
-
-/* eslint-enable id-length */
+/**
+ * The map tokens, mapped from the constants.
+ * @since 0.5.0
+ * @type {Map<string, Function>}
+ * @static
+ */
+Timestamp.mapTokens = new Map(Object.entries(TOKENS));
```

Comment:
What's the point of this being from Object.entries? It should also be `tokens` not `TOKENS` (latter is Record<string, number>)

```suggestion
Timestamp.mapTokens = new Map(tokens);
```

Reply:
`TOKENS`' type is `Record<string, number>` and maps stuff like `Y: 4` (max length for the token, since it can read up to `YYYY`), `mapToken`'s type is `Map<string, number>`, aka the same.

`tokens`' type is `Map<string, (time: Date) => string>`, they're not applicable.

Reply:
Three lines above... `* @type {Map<string, Function>}`
And how would that help us in any way, shape or form? We can already manually import the constants file and extract the tokens, while we cannot access the tokens Map you just implemented

Reply:
I guess I added it for consistency with the other classes, but I can 👢 it.